test/videojs-contrib-hls.test.js

```diff
@@ -2802,6 +2815,13 @@ QUnit.test('treats invalid keys as a key request failure and blacklists playlist
   hls.mediaSource.trigger('sourceopen');
   this.requests.shift()
     .respond(200, null,
+              '#EXTM3U\n' +
```

Comment:
(As per above): Why is this now responding with a master playlist? The URL above (provided as `src`) seems to indicate that it is a media playlist.